kolibri/plugins/learn/assets/src/views/SearchBox.vue

```diff
@@ -264,9 +264,9 @@
           'Learners can filter their searches for resources by type. For example, audio files, video files etc.',
       },
       topics: {
-        message: 'Topics',
+        message: 'Folders',
         context:
-          'Learners can filter their searches for resources by type. In this case, topics.\n\nA topic is a collection of resources and other topics within a channel.',
+          'Learners can also search for comlplete folders.\n\nA folder is a collection of resources and other subfolders within a channel.',
```

Comment:
"comlplete" >>> "complete"

Reply:
Spellchecker, spellchecker, spellchecker!!! 🤦🏽‍♀️ 
Thank you!